WeakAuras/WeakAuras2
# Improvements for custom options

# Description

Implements a few requests in #973

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Space height:
1. Add a space option. Configure the height option.
2. Note that in `User Mode`, the space obeys the height specified.

Duplication:
1. Add any option. Click the new button in the header
2. Note that a duplicate of the option you clicked the button on has been inserted directly below.

# Checklist:
